tests/unit/test_models.py

```diff
@@ -1,3 +1,10 @@
+<<<<<<< HEAD
```

Comment:
Need to remove the conflict comments

Reply:
ok, should be good now, lmk if I missed anything